src/ui-lib/sass/05-components/03-organisms/_hero.scss

```diff
@@ -1,21 +1,23 @@
 .grav-c-hero {
+  position: relative;
   padding: $grav-sp-xl 0;
-  background-blend-mode: hard-light;
-  background-color: $grav-co-accent-secondary-dark;
-  background-position-x: center;
-  background-size: cover;
+  background: linear-gradient(180deg, $grav-co-accent-secondary 0%, $grav-co-accent-secondary-darker 100%);
   color: $grav-co-bg;
   text-align: center;
 
-  @media all and (max-width: gravy-breakpoint(regular)) {
-    background-image: none !important;
-  }
-
   > * {
     @include grav-l-container;
   }
 
   > * + * {
     margin-top: $grav-sp-xl;
   }
+
+  > canvas {
+    max-width: initial;
+    position: absolute;
+    top: 0;
+    left: 0;
+    margin: 0;
```

Comment:
Need to add `width: 100%; height: 100%;` to ensure that the canvas always covers the entire hero section (which has an unpredictable width & height).